core/modules/system/views/views_handler_field_system_info.inc

```diff
@@ -0,0 +1,105 @@
+<?php
+/**
+ * @file
+ * Definition of views_handler_field_system_info.
+ */
+
+/**
+ * A handler to display module and theme settings stored in the info column.
+ *
+ * Provides access to several settings stored as a serialized array in the info
+ * column of the system table.
+ *
+ * @ingroup views_field_handlers
+ */
+class views_handler_field_system_info extends views_handler_field {
+  /**
+   * {@inheritdoc}
+   */
+  public function option_definition() {
+    $options = parent::option_definition();
+    $options['info_setting'] = array('default' => '');
+    $options['custom_label'] = array('default' => 'default');
+    return $options;
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  function options_form(&$form, &$form_state) {
+    parent::options_form($form, $form_state);
+
+    $form['custom_label']['#type'] = 'radios';
+    $form['custom_label']['#options'] = array(
+      'default' => t('Use setting label'),
```

Comment:
How about something like this?

```suggestion
      'default' => t('Same as in the .info file'),
```

Reply:
Changed the default with a more clear wording, though not exactly what you suggested.